libraries/AdaptiveExpressions/BuiltinFunctions/Rand.cs

```diff
@@ -43,9 +41,18 @@ private static EvaluateExpressionDelegate Evaluator()
                             }
                             else
                             {
-                                lock (_randomizerLock)
+                                if (options.RandomValue != null)
+                                {
+                                    value = options.RandomValue;
```

Comment:
>options.RandomValue; [](start = 44, length = 20)

This needs to be adjusted to be between min/max, i.e. min + (options.RandomValue % (max - min))